add project logo and art [SAME VERSION]

flynnduism opened the pull request:
This PR adds an /art directory containing brand assets and a readme with notes.

Signed-off-by: flynnduism dev@ronan.design

bfjelds commented:
a couple of the SVGs didn't look quite right in the Github viewer (https://github.com/deislabs/akri/blob/dce6668a99f391ac20cbb3b31e99212bf281ce68/art/icon/akri-icon-dark.svg and  https://github.com/deislabs/akri/blob/dce6668a99f391ac20cbb3b31e99212bf281ce68/art/icon/akri-icon-dark.png) ... i'm not sure if this is a problem with my browser or github or the images.

kate-goldenring commented on art/logo.svg:
```diff
@@ -0,0 +1 @@
+<svg height="1500" viewBox="0 0 3850 1500" width="3850" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><g fill-rule="nonzero"><g stroke="#444a6d" stroke-linecap="round" stroke-linejoin="round" stroke-width="44" transform="translate(369 269)"><path d="m845.181984 178.972621-310.939796 252.714879c5.399747 14.673932 9.162768 26.452578 11.289062 35.335937 2.126294 8.88336 4.009 ...
```
This is a Sonar logo

kate-goldenring commented:
[logo.png](https://github.com/deislabs/akri/blob/dce6668a99f391ac20cbb3b31e99212bf281ce68/art/logo.png) and [logo.svg](https://github.com/deislabs/akri/blob/dce6668a99f391ac20cbb3b31e99212bf281ce68/art/logo.svg) files are Sonar logos. were they accidentally included?

bfjelds commented:
It still looks like the patterns aren't quite right...they don't fill the sides when I look at this one: art/icon/akri-icon-dark.svg

flynnduism commented:
> It still looks like the patterns aren't quite right...they don't fill the sides when I look at this one: art/icon/akri-icon-dark.svg

Thanks @bfjelds, still a WIP to fix this. Redrawing part of the svg to avoid this distortion in the rendering - shall have updates shortly.

flynnduism commented:
> logo.png and logo.svg files are Sonar logos. were they accidentally included?

Yes 🤦 thanks @kate-goldenring for spotting that. Removed those.

---

Updated the `svg` files to fix the glitchy earlier rendering.

_Before:_

<img width="624" alt="Screen Shot 2020-11-25 at 4 08 17 PM" src="https://user-images.githubusercontent.com/686194/100293422-8b7f9380-2f38-11eb-9222-c9e94293bfa4.png">

_Now:_

<img width="629" alt="Screen Shot 2020-11-25 at 4 08 49 PM" src="https://user-images.githubusercontent.com/686194/100293428-92a6a180-2f38-11eb-9d76-4c4f66d06593.png">

bfjelds commented:
Awesome!  Almost all of the images look right to me now.  Only art/logo.svg looks off to me now.

flynnduism commented:
> Awesome! Almost all of the images look right to me now. Only art/logo.svg looks off to me now.

Thanks @bfjelds that's updated now

kate-goldenring commented on art/README.md:
````diff
@@ -0,0 +1,49 @@
+Brand only (not for text):
+
+```
+$purplebrand:   #78FFC9;
````
I think the hex for these is swapped. I haven't checked the others, though

flynnduism commented:
Some follow-up adjustments to the branding to consider here:

# Logo Wordmark

<img width="550" alt="Screen Shot 2021-01-12 at 11 26 39 AM" src="https://user-images.githubusercontent.com/686194/104362830-2290b280-54c9-11eb-91d9-0192e9c5ed32.png">

<img width="550" alt="Screen Shot 2021-01-12 at 11 26 39 AM" src="https://user-images.githubusercontent.com/686194/104362966-58359b80-54c9-11eb-83c8-7c4849d173c1.png">

I've had feedback that the above typeface style for `Akri` isn't a good fit; to explore a more developer-y, monospaced approach instead of the rounded, playful font originally included in this PR.
 
## Some Alternatives

<img width="508" alt="Screen Shot 2021-01-11 at 10 27 10 PM" src="https://user-images.githubusercontent.com/686194/104362994-64b9f400-54c9-11eb-8f04-5ef0f2f5f252.png">

Here are some explorations of different styles for this - ranging from typical monospaced typefaces, to some that are slightly curved / chunkier / more unusual. 

## Pairing Type + Icon

I think these ones in particular stand out:

<img width="221" alt="Screen Shot 2021-01-11 at 10 28 05 PM" src="https://user-images.githubusercontent.com/686194/104363155-a34fae80-54c9-11eb-9dd6-83f003a623e3.png">

_PT Mono is a nice, blocky one with a line thickness that works well with the line styles in the icon._ 

<img width="236" alt="Screen Shot 2021-01-11 at 10 28 19 PM" src="https://user-images.githubusercontent.com/686194/104363238-c37f6d80-54c9-11eb-99ed-600d29ee76fb.png">

_Input is one of my favorites, and looks nice in an uppercase 'stamp' style._ 

<img width="258" alt="Screen Shot 2021-01-11 at 10 28 14 PM" src="https://user-images.githubusercontent.com/686194/104363322-dc881e80-54c9-11eb-8984-f7ffacae4b4b.png">

_Major Display Mono is a very fun one, has a clean sci-fi style. Looks striking. The lines are very thin though which can be a problem at smaller sizes (no bold option)._

flynnduism commented:
I'd like to get some feedback on these approaches, to figure out the best branding style to use here. Any input appreciated!

kate-goldenring commented:
Thanks @flynnduism for sharing all these font options. I think I like Input Mono the best. In general, I like the log with uppercase A and the rest of the letters lowercased. I also wonder if a font without the foot on the i would look good like this one you tried out at one point:
![image](https://user-images.githubusercontent.com/19913192/104378289-fe42cf00-54e4-11eb-8718-7695fce49c25.png)

Or maybe the foot on the i solidified the end of the name and its intentional? I'm definitely not familiar with the ways of branding. 

Major display is a little too creative for me.

flynnduism commented:
Thanks Kate, I really like Input too (it's very [thoughtfully created](https://input.fontbureau.com/info/) for developers)!

Input in different weights:

![akri-input-standard](https://user-images.githubusercontent.com/686194/104503521-060e7c00-5596-11eb-8f2a-352c72f69c56.png)


> I also wonder if a font without the foot on the i would look good like this one you tried out at one point

Good idea. This style where the "i" has a foot is sort of a staple characteristic of the monospaced look... the closest I could find to a monospaced type style without is [source code pro](https://adobe-fonts.github.io/source-code-pro/) (from adobe). Here I've customized Input to explore that look:

![akri-input-custom](https://user-images.githubusercontent.com/686194/104503547-0eff4d80-5596-11eb-809f-539e6c8ea394.png)

edrickwong commented:
I think from the alternative fonts list, Inconsolata (with the regular casing) has a nice balance between being more developer-y without going totally monospaced. I would be curious to see if there are other weightings of it, but overall I think the font itself is a good one to go with.

kate-goldenring commented:
> I think from the alternative fonts list, Inconsolata (with the regular casing) has a nice balance between being more developer-y without going totally monospaced. I would be curious to see if there are other weightings of it, but overall I think the font itself is a good one to go with.

Good point, I like how the sharpness of the A of Inconsolata works with the logo. Inconsolata works for me, too.

flynnduism commented:
Ok awesome. Here's a comparison of Inconsolata weights:

![akri-inconsolata-weights](https://user-images.githubusercontent.com/686194/104632979-54ce1b80-5653-11eb-8e11-2fa98982f0ff.png)

My favorite is bold, where the line weight feels like a good match for the icon, but the letters still have fairly clean corners.

romoh commented:
> Ok awesome. Here's a comparison of Inconsolata weights:
> 
> ![akri-inconsolata-weights](https://user-images.githubusercontent.com/686194/104632979-54ce1b80-5653-11eb-8e11-2fa98982f0ff.png)
> 
> My favorite is bold, where the line weight feels like a good match for the icon, but the letters still have fairly clean corners.

+1 on bold.
The line weight matches the icon lines and pairs well.

flynnduism commented:
This PR has been updated so that the brand assets and readme notes are now all using Inconsolata Bold. Ready for review.

<img width="559" alt="akri-logos" src="https://user-images.githubusercontent.com/686194/104776503-0c842b80-572f-11eb-8c69-a74df6d3d3f3.png">

kate-goldenring commented on art/README.md:
```diff
@@ -0,0 +1,52 @@
+# AKRI Logo & Assets
```
"Akri" is foremost a proper noun -- means edge in Greek -- and secondly also conveniently and acronym (AKRI). Certainly confusing but possibly twice the fun. Could you update the documentation here to use "Akri"?

flynnduism replied:
Thanks @kate-goldenring I've corrected the casing in the readme.

kate-goldenring commented on art/README.md:
````diff
@@ -0,0 +1,52 @@
+$greenshade:    #8CCFCE;
+$purpleshade:   #524886;
+```
+
````
Should the hex for the accent palette be added here?

flynnduism replied:
Good call - added these in!

romoh commented:
> This PR has been updated so that the brand assets and readme notes are now all using Inconsolata Bold. Ready for review.
> 
> <img alt="akri-logos" width="559" src="https://user-images.githubusercontent.com/686194/104776503-0c842b80-572f-11eb-8c69-a74df6d3d3f3.png">

@flynnduism : A minor comment, are the colors used in the dark version the same as the light version? The colors of the two sides of the cube in the dark version seem to be almost the same (greenish). they appear to have less contrast, or is that the magic of how our eyes handle change in the background?

flynnduism commented:
> > This PR has been updated so that the brand assets and readme notes are now all using Inconsolata Bold. Ready for review.
> > 
> > <img alt="akri-logos" width="559" src="https://user-images.githubusercontent.com/686194/104776503-0c842b80-572f-11eb-8c69-a74df6d3d3f3.png">
> 
> @flynnduism : A minor comment, are the colors used in the dark and light versions the same? The colors of the two sides of the cube in the dark version looks very similar (greenish). they appear to have less contrast, or is that the magic of how our eyes handle change in the background?

@romoh good eyes! Yes this is intentional, the colors have variants depending on whether the logo is in a light setting or a dark setting, to ensure contrast and legibility ar maintained.

Hence this PR contains two sets of assets to be used accordingly for light and dark modes, the idea being that the branding has [adaptive colors](https://medium.com/eightshapes-llc/light-dark-9f8ea42c9081) because how we perceive color changes depending on the light/dark context.

flynnduism commented:
Woo hoo, thanks @kate-goldenring!
